api/v01/api.rb

```diff
@@ -133,8 +133,6 @@ def count_incr(operation, options)
           rack_response(format_message(response, nil), 417)
         elsif e.is_a?(RouterWrapper::InvalidArgumentError)
           rack_response(format_message(response, nil), 400)
-        elsif e.is_a?(Wrappers::UnreachablePointError)
```

Comment:
Can you also delete the `Wrappers::UnreachablePointError` class, after the cleaning there is references to it.

Reply:
Yes, done.